xAPI.md

```diff
@@ -216,17 +216,22 @@ JSON allows for objects to have properties that contain empty objects. This is n
 * [Actor](#def-actor)
 * [Authentication](#def-authentication)
 * [Authorization](#def-authorization)
-* [Base Endpoint](#def-baseendpoint)
+* [Base Resource Path](#def-baseresourcepath)
 * [Community of Practice](#def-community-of-practice)
-* [Experience API (xAPI)](#def-experience-api)
+* [Experience Application Programming Interface (xAPI)](#def-experience-api)
```

Comment:
I'm fine with adding a definition for API if you want to, though it should be short and reference an external source.